demos/serverless/src/handlers.js

```diff
@@ -341,16 +342,98 @@ exports.end_capture = async (event, context) => {
     }).promise();
     return response(200, 'application/json', JSON.stringify({}));
   } else {
-    return response(500, 'application/json', JSON.stringify({ msg: "No pipeline to stop for this meeting" }))
+    return response(500, 'application/json', JSON.stringify({msg: "No pipeline to stop for this meeting"}))
+  }
+};
+
+
+exports.start_live_connector = async (event, context) => {
+  // Fetch the meeting by title
+  const meeting = await getMeeting(event.queryStringParameters.title);
+  meetingRegion = meeting.Meeting.MediaRegion;
+  const ivsChannelNamePrefix = IVS_CHANNEL_PREFIX?IVS_CHANNEL_PREFIX: "liveConnector"
+  const ivsChannelName = `${ivsChannelNamePrefix}-${meetingRegion}`;
+  ivsEndpoint = ''
+  rtmpURL = ''
+  ivsPlaybackUrl = ''
+  ivsChannelArn = ''
+  const params = {
+    name: ivsChannelName,
+  };
+  const ivsCreateChannelResponse = await ivs.createChannel(params).promise();
```

Comment:
Why do we create the IVS channel in serverless demo but not locally in `server.js`?

Reply:
in serverless, we don't need to create the ivs channel. IVS channel will be created when we hit the button. But server.js is expect user to have ivs channel created. User need to add the rtmp url in the environment variable.

Reply:
Actually we want the experience to be the same in both local and serverless demos no?

Reply:
Discussed offline. For local demos, will just use environment variable.